### ISSUE TITLE
Train Model

### ISSUE THREAD

**Issue by tholoien** [tholoien/empiriciSN#12]
Split the SN sample into training and validation sets and use this to train the model. Use cross-validation to determine the right number of parameters to incorporate and measure the accuracy of the model.

**Comment by drphilmarshall** [comment]
When we met with Risa, we discussed the following splits:
1. No split: all SNe analysed as a single population
2. Exp hosts vs DeV hosts
3. High z vs Low z
4. SDSS vs SNLS

To assess significant differences between the resulting models, we'll need
a way of quantifying uncertainty in the GMM parameters: we decided we'd
need jackknife or bootstrap, if we go with the astroML XDGMM...

**Comment by drphilmarshall** [comment]
Looks like this issue can be closed, @tholoien ! You can re-issue the CV optimization separately, but for now you have a trained model that can be used to explore the conditional PDFs to allow SNe to be predicted in new (test set) host galaxies.
